package-lock.json

```diff
@@ -1306,7 +1306,7 @@
     },
     "buffer-equal-constant-time": {
       "version": "1.0.1",
-      "resolved": "https://registry.npmjs.org/buffer-equal-constant-time/-/buffer-equal-constant-time-1.0.1.tgz",
+      "resolved": "https://workable.jfrog.io/workable/api/npm/npm/buffer-equal-constant-time/-/buffer-equal-constant-time-1.0.1.tgz",
```

Comment:
❓ Was this intentionally changed?

Reply:
Hm.. Since this is a public open-source project I should probably revert this